spacy_llm/pipeline/llm.py

```diff
@@ -238,4 +249,16 @@ def from_disk(
         exclude (Tuple): Names of properties to exclude from deserialization.
         RETURNS (LLMWrapper): Modified LLMWrapper instance.
         """
+
+        serialize = {}
+
+        task = self._task
+        backend = self._backend
```

Comment:
Why introduce aliases here?

Reply:
I was trying to fool the static type checker, but that was no remedy... I'll remove those, thanks.